model/model_training/configs/config.yaml

```diff
@@ -354,6 +356,78 @@ llama-30b-pretrain:
   num_train_epochs: 1
   save_total_limit: 2
 
+lora-llama-13b:
```

Comment:
Should this be lora-llama-7b?

Reply:
No sorry, I just didn't bother changing the logging dirs. Will change the log_dir names.

Reply:
it is meant to be 13B . you can fit much higher bs. the model_name is a mistake (I had some code to use the 7B model as a tokenizer for all models, and download the weights from HF)